lib/transform.js

```diff
@@ -99,17 +101,19 @@ function transform(source, template, options = {}) {
   return { source, template };
 }
 
-function guessTemplatePath(componentPath) {
+function guessPossibleTemplatePaths(componentPath) {
```

Comment:
What about:
```suggestion
function possibleTemplatePaths(componentPath) {
```

Reply:
Another concern, is the old method part of the public API of the package? I was going to suggest a deprecation path here, but the package is not 1.0 yet 🤔

Reply:
this package does not have a public API. to only "public API" is the CLI.

Reply:
Done!